CODE/VulnScan/tools/_test_gpu_acceleration.py

```
@@ -0,0 +1,24 @@
+try:
+    # noinspection PyUnresolvedReferences
+    import torch
+except ImportError as e:
+    print(f"Error: Failed to import torch. Please ensure PyTorch is installed correctly: {e}")
+    exit(1)
+
+
+def check_gpu():
```

Ruff (0.8.0)

Missing return type annotation for private function `check_gpu`

Add return type annotation: `None`

(ANN202)